test-lib.sh

```diff
@@ -252,7 +253,7 @@ function ct_npm_works() {
     return 1
   fi
 
-  docker run $(ct_mount_ca_file) --rm ${IMAGE_NAME} /bin/bash -c "npm install jquery && test -f node_modules/jquery/src/jquery.js"
+  docker exec $(cat ${cid_file}) /bin/bash -c "npm install jquery && test -f node_modules/jquery/src/jquery.js"
```

Comment:
Not sure I understand this change. You will need to run the container somewhere before you are able to exec into it.

Reply:
After discussing this online and suggesting a different way to create the files needed for npm to use custom registries it hit me that this might still not work because of https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L554 only being run during a s2i build :|

Reply:
Right, I have to do it in a different way.